images/linux/scripts/installers/mysql.sh

```diff
@@ -9,19 +9,30 @@ source $HELPER_SCRIPTS/document.sh
 
 export ACCEPT_EULA=Y
 
-# Install MySQL Client
-apt-get install mysql-client -y
+# Mysql root password
+MYSQL_ROOT_PASSWORD=root
 
-# InstallMySQL database development files
-apt-get install libmysqlclient-dev -y
+if isUbuntu16 || isUbuntu18 ; then
+    apt-get install mysql-client -y
+fi
 
-# Install MySQL Server
-MYSQL_ROOT_PASSWORD=root
-echo "mysql-server mysql-server/root_password password $MYSQL_ROOT_PASSWORD" | debconf-set-selections
-echo "mysql-server mysql-server/root_password_again password $MYSQL_ROOT_PASSWORD" | debconf-set-selections
```

Comment:
Looks like before your changes these lines were invoked on Ubuntu 18 but after your changes that are not invoked. Is it expected?

Reply:
Thank you, you are absolutely right, I moved this part accidentally.